subgraphs/uniswap-v3/src/common/dex_event_handler.ts

```diff
@@ -582,6 +576,20 @@ export class DexEventHandler {
     this.pool.save();
   }
 
+  updateAndSavePoolTokenEntities(): void {
+    for (let i = 0; i < this.poolTokens.length; i++) {
+      const poolToken = this.poolTokens[i];
```

Comment:
I think this array will automatically get sorted since is an array of entities: https://github.com/messari/subgraphs/blob/master/docs/ERRORS.md#array-sorting-when-querying

Do we know poolTokens and inputTokenBalanceDeltas are parallel?

Reply:
Yes, this will be the case, I always and handling the tokens in the object with the same ordering on the pool, which matches the ordering from the contracts (amount0, amount1)